### PR TITLE
Introduce an events limit for the Profit loss report for free users

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

**Comment by kelsos** [comment]
Ok so need to modify the component to support this use case. This component is the same used in the other tables with similar functionality.

File: frontend/app/src/locales/en.json [hunk 1]
```diff
@@ -1861,7 +1873,8 @@
     "loading_hint": "Your report generation might take a while depending on the amount of trades and actions you performed during the selected period.",
     "select_directory": "Select a directory",
     "csv_export_error": "There was an error with Export",
-    "report_period": "Report starting from {start} to {end}"
+    "report_period": "Report starting from {start} to {end}",
+    "upgrade": "{processed} total events have been processed starting from {start}. The limit of the free version as been reached. To get the full report please upgrade to {link}"
```
Comment (LefterisJP) [inline frontend/app/src/locales/en.json:1877]:
```suggestion
    "upgrade": "{processed} total events have been processed starting from {start}. The limit of the free version has been reached. To get the full report please upgrade to {link}"
```

**Review by LefterisJP — COMMENTED** [review]
Looks good.

I think I have an off by 1 error in the backend I can fix in another PR.

Leaving one typo suggestion and a question/request.

Can we move the message inside the box, or in its own box somehow? It looks a bit out of place where it is.

**Comment by kelsos** [comment]
> Leaving one typo suggestion and a question/request.
> 
> Can we move the message inside the box, or in its own box somehow? It looks a bit out of place where it is.
> 
> 

own box then it doesn't make sense to add it in the accounting settings display

**Comment by LefterisJP** [comment]
Looks good to me. I checked screenshot from kelsos. The failing test is the kucoin sandbox trades which we don;t know yet why it fails, while it used to pass before when the kucoin pr got merged.

**Pull request merged**